static/app/views/onboarding/onboardingController.tsx

```diff
@@ -9,10 +9,12 @@ import Onboarding from './onboarding';
 type Props = Omit<ComponentPropsWithoutRef<typeof Onboarding>, 'projects'>;
 
 function OnboardingController({...rest}: Props) {
-  logExperiment({
-    key: 'TargetedOnboardingMultiSelectExperiment',
-    organization: rest.organization,
-  });
+  useEffect(() => {
+    logExperiment({
+      key: 'TargetedOnboardingMultiSelectExperiment',
+      organization: rest.organization,
+    });
+  }, []);
```

Comment:
@Neo-Zhixing I would just omit the second arg here to make the warning message go way

Reply:
But then wouldn't this fire a lot more frequently

Reply:
@Neo-Zhixing It's only going to fire when `rest.organization` changes. If the org changes, we would want this to fire again anyways (in practice that's probably impossible but it's less code this way too).